suite-native/atoms/src/CardStepper/CardStepper.tsx

```diff
@@ -17,8 +18,8 @@ export type CardStepperMap<ContentIdType = undefined> = Record<
 
 type CardStepperProps<ContentIdType = undefined> = {
     onFinish: () => void;
-    primaryButtonText: ReactNode;
-    secondaryButtonText: ReactNode;
+    primaryButtonText: TxKeyPath;
```

Comment:
Just food for thought, no need to change it - I believe it's almost always better to pass the Translation component instead of just the translation key, because it's more flexible. I prefer to pass only the translation key if the component has to be sure that it's only text (because the component is styling that text globally or something like that).